elogia_hr/models/hr.py

```diff
@@ -40,18 +52,352 @@ class EmployeeHistory(models.Model):
+        ('processed', 'Processed'),
+    ], string='Status', default='not_processed', required=True)
+    list_ids = fields.One2many('list.action.change', 'planning_id', 'List Action')
+    check_company = fields.Boolean('Company?')
+    process_company = fields.Boolean('Process Company')
+    company_id = fields.Many2one('res.company', 'Company', ondelete='restrict')
+    department_id = fields.Many2one('hr.department', 'Department', ondelete='restrict')
+    check_department = fields.Boolean('Department?')
+    process_department = fields.Boolean('Process Department')
+    parent_id = fields.Many2one('hr.employee', 'Responsible', ondelete='restrict')
+    check_parent = fields.Boolean('Responsible?')
+    process_parent = fields.Boolean('Process Parent')
+    hub_id = fields.Many2one('planning.role', 'Rol/Hub', ondelete='restrict')
+    check_hub = fields.Boolean('Rol/Hub?')
+    process_hub = fields.Boolean('Process Hub')
+    job_id = fields.Many2one('hr.job', 'Job', ondelete='restrict')
+    check_job = fields.Boolean('Job?')
+    process_job = fields.Boolean('Process Job')
+    structure_type_id = fields.Many2one('hr.payroll.structure.type', 'Contratation Type', ondelete='restrict')
+    check_structure = fields.Boolean('Contratation Type?')
+    process_structure = fields.Boolean('Process Structure')
+    contract_type_id = fields.Many2one('hr.contract.type', 'Type Contract', ondelete='restrict')
+    check_contract = fields.Boolean('Type Contract?')
+    process_contract = fields.Boolean('Process Contract')
+    scholarship_id = fields.Many2one('type.scholarship', 'Type Scholarship', ondelete='restrict')
+    check_scholarship = fields.Boolean('Type Scholarship?')
+    process_scholarship = fields.Boolean('Process Scholarship')
+    departure_id = fields.Many2one('hr.departure.reason', 'Departure Reason', ondelete='restrict')
+    check_departure = fields.Boolean('Departure Reason?')
+    process_departure = fields.Boolean('Process Departure')
+    departure_date = fields.Date('Departure Date')
+    check_departure_date = fields.Boolean('Departure Date?')
+    process_departure_date = fields.Boolean('Process Departure Date')
+    wage = fields.Float('Fixed wage', tracking=True, help="Employee's annually gross wage fixed.")
+    check_wage = fields.Boolean('Wage?')
+    process_wage = fields.Boolean('Process Wage')
+    wage_variable = fields.Float('Variable Wage', tracking=True, help="Employee's annually gross wage variable.")
+    check_wage_variable = fields.Boolean('Wage V.?')
+    process_wage_v = fields.Boolean('Process Wage V')
+    currency_id = fields.Many2one('res.currency', compute='_get_company_currency', readonly=True, string="Currency",
+                                  help='Utility field to express amount currency')
+
+    def action_open_wizard(self):
+        return {
+            'type': 'ir.actions.act_window',
+            'name': 'Apply changes for %s.' % self.employee_id.name,
```

Comment:
es mas recomendable el uso de la funcion "format" para el manejo de string con python. Ejemplo
'Apply changes for {}.' .format(self.employee_id.name)

Reply:
Listo